test/distributed/fsdp/test_fsdp_mixed_precision.py

```diff
@@ -860,9 +883,14 @@ def test_full_precision_in_eval_buffers(self):
             # model.eval() + forward pass should make the buffers in full prec again
             fsdp_model.eval()
             fsdp_model((inp, self, fsdp_model, mp_config, torch.float32))
-
+            # TODO: this test would be more robust if the buffer dtype was
```

Comment:
Make this a bootcamp/onboarding task? :)

Reply:
https://github.com/pytorch/pytorch/issues/104740